clients/acm.d.ts

```diff
@@ -0,0 +1,350 @@
+   */
+  requestCertificate(params: ACM.RequestCertificateRequest, callback?: (err: AWSError, data: ACM.RequestCertificateResponse) => void): Request<ACM.RequestCertificateResponse, AWSError>;
+  /**
+   * Requests an ACM Certificate for use with other AWS services. To request an ACM Certificate, you must specify the fully qualified domain name (FQDN) for your site. You can also specify additional FQDNs if users can reach your site by using other names. For each domain name you specify, email is sent to the domain owner to request approval to issue the certificate. After receiving approval from the domain owner, the ACM Certificate is issued. For more information, see the AWS Certificate Manager User Guide .
+   */
+  requestCertificate(callback?: (err: AWSError, data: ACM.RequestCertificateResponse) => void): Request<ACM.RequestCertificateResponse, AWSError>;
+  /**
+   * Resends the email that requests domain ownership validation. The domain owner or an authorized representative must approve the ACM Certificate before it can be issued. The certificate can be approved by clicking a link in the mail to navigate to the Amazon certificate approval website and then clicking I Approve. However, the validation email can be blocked by spam filters. Therefore, if you do not receive the original mail, you can request that the mail be resent within 72 hours of requesting the ACM Certificate. If more than 72 hours have elapsed since your original request or since your last attempt to resend validation mail, you must request a new certificate.
+   */
+  resendValidationEmail(params: ACM.ResendValidationEmailRequest, callback?: (err: AWSError, data: {}) => void): Request<{}, AWSError>;
+  /**
+   * Resends the email that requests domain ownership validation. The domain owner or an authorized representative must approve the ACM Certificate before it can be issued. The certificate can be approved by clicking a link in the mail to navigate to the Amazon certificate approval website and then clicking I Approve. However, the validation email can be blocked by spam filters. Therefore, if you do not receive the original mail, you can request that the mail be resent within 72 hours of requesting the ACM Certificate. If more than 72 hours have elapsed since your original request or since your last attempt to resend validation mail, you must request a new certificate.
+   */
+  resendValidationEmail(callback?: (err: AWSError, data: {}) => void): Request<{}, AWSError>;
+}
+declare namespace ACM {
```

Comment:
I wanted to export all the interfaces so that users could cast a result if they needed to, but I'm not sure if that's necessary. I'll have to take a look at some other TypeScript libraries to see if that's common practice.

You should only see these interfaces when looking at the service client constructors, but not on the service client instances.